PWGCF/Correlations/DPhi/AliAnalysisTaskPhiCorrelations.cxx

```diff
@@ -2051,6 +2058,17 @@ Bool_t AliAnalysisTaskPhiCorrelations::InitiateEventPlane(Double_t& evtPlanePhi,
 }
 
 //____________________________________________________________________
+Long64_t AliAnalysisTaskPhiCorrelations::GetUniqueEventID(AliVEvent* inputEvent)
+{
+  // Get event ID from header
+  AliVHeader* eventIDHeader = inputEvent->GetHeader();
+  if(eventIDHeader)
+    return eventIDHeader->GetEventIdAsLong();
+  else
```

Comment:
I thought since the ID is calculated using BC, period, and orbit, it is kind of global.
http://aliroot-docs.web.cern.ch/aliroot-docs/src/AliVHeader.cxx.html#49

Maybe I am wrong and you know the correct code to get a global event ID?